andela/ah-backend-athena
# #163183529 Fix search functionality

**What does this PR do?**
This PR fixes the search functionality feature to be able to return all fields about an article

**How can this be manually tested?**
Navigate to the endpoint for returning all articles ie `api/articles/search` and you can add on the following search queries
 - `api/articles/search?tag=magic` to see all articles with the tag `magic`
- `api/articles/search?author=ntale` to see all articles authored by `ntale`
- `api/articles/search?keyword=dragon` to see all articles with the word  `dragon` in their title

**Before**
<img width="1108" alt="screenshot 2019-01-14 at 11 02 28" src="https://user-images.githubusercontent.com/39732510/51101520-f1068300-17eb-11e9-87ea-babf3140d4c0.png">

**After**
<img width="1109" alt="screenshot 2019-01-30 at 09 58 09" src="https://user-images.githubusercontent.com/39732510/51963777-a8023000-2475-11e9-80a1-8555bea09ed8.png">
